.gitignore

```diff
@@ -2,8 +2,8 @@ coverage
 npm-debug.log
 node_modules
 build
-dist
-lib
+#dist
+#lib
```

Comment:
Had to unignore these so that they'd be checked in because I'm sourcing the npm module from git.